src/object.js

```diff
@@ -39,7 +40,8 @@ define([
 	 * Creating instances:
 	 *
 	 *    var milou = dog({name: milou});
-	 *    milou.say('hello Tintin'); ;; => 'Woof Woof, hello Tintin'
+	 *    console.log(milou.say('hello Tintin')); ;; => 'Woof Woof, hello Tintin'
```

Comment:
@DamienCassou it's quite common in JS examples to omit the `console.log` for readability.
The value in the comment is the value returned, not printed